image_captcha/image_captcha.admin.inc

```diff
@@ -1,4 +1,8 @@
 <?php
+/**
+ * @file
+ * Contains \Drupal\captcha\image_captcha\image_captcha.admin.inc.
```

Comment:
The Contains line is only for classes. A file containing file is recursive, it's also has nothing to do with namespaces :)

Say something like "Contains functions used in the backend forms." or so. it will go away anway...